core/js/src/typespecs/api_types.ts

```diff
@@ -330,6 +330,7 @@ const experimentEventSchema = z
       "A dictionary of numeric values (between 0 and 1) to log. The scores should give you a variety of signals that help you determine how accurate the outputs are compared to what you expect and diagnose failures. For example, a summarization app might have one score that tells you how accurate the summary is, and another that measures the word similarity between the generated and grouth truth summary. The word similarity score could help you determine whether the summarization was covering similar concepts or not. You can use these scores to help you sort, filter, and compare experiments"
     ),
     metadata: experimentEventBaseSchema.shape.metadata,
+    tags: z.array(z.string()).nullish().describe("A list of tags to log"),
```

Comment:
Add this to `generateBaseEventOpSchema`, so it can be shared across all the object types.